package.json

```diff
@@ -2,10 +2,11 @@
   "name": "AMS",
   "private": true,
   "engines": {
-    "node": "12.22.10"
```

Comment:
@alishaevn is this going to cause problems?

Reply:
I don't believe so. we changed it because it was causing the github actions in this repo to fail.

talked about it [here](https://assaydepot.slack.com/archives/C030UPFFP2A/p1646684584156969?thread_ts=1646684579.748519&cid=C030UPFFP2A).